docs/jwst/ramp_fitting/main.rst

```diff
@@ -110,8 +110,8 @@ optimal weighting. The variance of the slope of the segment due to read noise is
 .. math::  
    var^R_{s} = \frac{12 \ R^2 }{ (ngroups_{s}^3 - ngroups_{s})(tgroup^2) } \,,
 
-\noindent where $R$ is the noise in the difference between 2 frames, 
-$ngroups_{s}$ is the number of groups in the segment, and $tgroup$ is the group 
+\noindent where :math:`R` is the noise in the difference between 2 frames, 
```

Comment:
There are three places where you used "\noindent".  These show up as the actual word "noindent" in the text.